sermant-backend/src/main/java/io/sermant/backend/service/MountService.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.backend.service;
+
+import io.sermant.backend.entity.config.Result;
+import io.sermant.backend.entity.config.ResultCodeType;
+import io.sermant.backend.entity.mount.CommandType;
+import io.sermant.backend.entity.mount.MountConfig;
+import io.sermant.implement.service.dynamicconfig.ConfigClient;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Service;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.annotation.Resource;
+
+/**
+ * Mount Service
+ *
+ * @author zhp
+ * @since 2024-08-22
+ */
+@Service
+public class MountService {
+    private static final String MOUNT_CONFIG_KEY = "mount-config";
+
+    private static final String MOUNT_CONFIG_GROUP = "sermant-dynamic-mount";
+
+    @Resource
+    private ConfigService configService;
+
+    private final Yaml yaml = new Yaml();
+
+    /**
+     * publish configuration
+     *
+     * @param mountConfig mount configuration information
+     * @return The result of publish mount configuration
+     */
+    public Result<Boolean> publishMountConfig(MountConfig mountConfig) {
+        if (!validateParam(mountConfig)) {
+            return new Result<>(ResultCodeType.MISS_PARAM);
+        }
+        ConfigClient configClient = configService.getConfigClient();
+        boolean result = configClient.publishConfig(MOUNT_CONFIG_KEY, MOUNT_CONFIG_GROUP, yaml.dumpAsMap(mountConfig));
+        if (result) {
+            return new Result<>(ResultCodeType.SUCCESS);
+        }
+        return new Result<>(ResultCodeType.FAIL);
+    }
+
+    /**
+     * validate Param
+     *
+     * @param mountConfig mount configuration information
+     * @return verification results
+     */
+    private boolean validateParam(MountConfig mountConfig) {
+        return StringUtils.isNotEmpty(mountConfig.getPluginNames())
+                || !(StringUtils.equals(mountConfig.getCommandType(), CommandType.UPDATE_PLUGINS.getValue())
+                || StringUtils.equals(mountConfig.getCommandType(), CommandType.UNINSTALL_PLUGINS.getValue()));
+    }
```

Comment:
UPDATE_PLUGINS is invalid?

Reply:
This method is mainly to judge plugin name cannot be empty during plug-in installation, uninstallation, or update。UPDATE_PLUGINS is valid